javalib/src/main/scala/java/net/InetAddress.scala

```diff
@@ -88,8 +90,8 @@ class InetAddress protected (ipAddress: Array[Byte], originalHost: String)
     if (ipAddress.length == 4) {
       formatIn4Addr(bytes)
     } else if (ipAddress.length == 16) {
-      if (isIPv4MappedAddress(bytes)) {
-        formatIn4Addr(extractIP4Bytes(bytes).at(0))
+      if (SocketHelpers.isIPv4MappedAddress(bytes)) {
```

Comment:
Why this change??

Reply:
I copied the entire directory of this PR to my system, so that I could grep for usages.
Before this PR, `isIPv4MappedAddress()` is used in only one file `InetAddress.scala`, hence declared as `local`.

This PR moves it to `SocketHelpers` but, if I am searching correctly, still only used in one file.

What am I missing? Seems to me that the principle of "least change"  would suggest that two
files which do not have UDP related changes should not be touched/changed.  I will
admit that there are exceptions to that principle for flagrant and egregious defects.

I do concur that a usage in a second or additional files would be a good cause to move
to `SocketHelpers`.  

I did not review the moved code in detail, GitHub display adds a lot of noise. Is it
otherwise unchanged and bug for bug identical?